src/mixins/filter-mixin.js

```diff
@@ -15,17 +15,56 @@ export function addFilterHandler(filters, filter) {
   return filters
 }
 
-export function hasFilterHandler(filters, filter) {
-  if (typeof filter === "undefined") {
+/**
+ * hasFilterHandler
+ * - if testValue is undefined, checks to see if the chart has any active filters
+ * - if testValue is defined, checks to see if that testValue passes the active filter
+ *
+ * @param {*} filters - the chart's current filter
+ * @param {*} testValue - a value being tested to see if it passes the filter
+ *
+ *  - If chart values are not binned:
+ *      - Params will be an array of values
```

Comment:
Does "params" here mean `testValue`?

Reply:
Both params - filters, and testValue

Reply:
So are `filters` and `testValues` always the same structure (either a plain array, or an array of arrays)? Or can one be a regular array while the other is a array of arrays?

Reply:
I only looked into a couple test cases, but that's how it appeared. I added some uncertainty to my documentation just in case